cairo/kakarot-ssj/crates/evm/Scarb.toml

```
@@ -9,8 +9,8 @@ edition = "2024_07"
 starknet.workspace = true
 utils = { path = "../utils" }
 contracts = { path = "../contracts" }
-openzeppelin = { path = "../openzeppelin" }
 garaga = { git = "https://github.com/keep-starknet-strange/garaga.git" }
+openzeppelin = { git = "https://github.com/openzeppelin/cairo-contracts", tag="v0.16.0" }
```

```suggestion
openzeppelin = { git = "https://github.com/openzeppelin/cairo-contracts", tag="v0.18.0" }
```

same, perhaps we can use 0.18.0 ?